package/MDAnalysis/auxiliary/__init__.py

```diff
@@ -494,16 +494,16 @@
 
   ``close()``
 
-For convinience, when reading auxiliary data from an open file, step at a time, 
-:class:`~MDAnalysis.auxiliary.base.AuxFileReader` 
+For convenience, when reading auxiliary data from an open file, step at a time,
+:class:`~MDAnalysis.auxiliary.base.AuxFileReader`
 extends :class:`~MDAnalysis.auxiliary.base.AuxReader` by providing the
 following (these may be overwritten by subclasses as appropriate):
 
   ``__init__(auxfile, **kwargs)``
     Open *auxfile* and any additional processing of *kwargs*.
 
-  ``_restart()``, 
-    Seek to the start of ``auxfile``
+  ``_restart()``,
+  Seek to the start of ``auxfile``
```

Comment:
```suggestion
    Seek to the start of ``auxfile``
```

The spaces removed here were important for the formatting -- see https://mdanalysis--3646.org.readthedocs.build/en/3646/documentation_pages/auxiliary/init.html